docs/src/modules/utils/useCustomizationPlayground.tsx

```diff
@@ -0,0 +1,355 @@
+    }
+    return <Component {...props} />;
+  };
+}
+
+interface Props
+  extends Pick<
+      UseCustomizationPlaygroundReturnType,
+      'selectedDemo' | 'selectedSlot' | 'selectedCustomizationOption' | 'selectedTokens'
+    >,
+    Pick<UseCustomizationPlaygroundProps, 'componentName'> {
+  theme: Theme;
+  examples: CustomizationItemType;
+}
+
+/* I use this method to parse whatever component props are passed in and format them for the code example, 
+so the code example includes the same props as the rendered component. e.g. the views={['month']} */
+function formatComponentProps(componentProps?: Object, spacing: number = 1) {
+  function formatObject(obj: Object, indentLevel = 0, separator = ': '): string {
+    const indent = ' '.repeat(indentLevel * 2);
+
+    return (Object.keys(obj) as Array<keyof typeof obj>)
+      .map((key) => {
+        const getValue = (val: any) => {
+          if (typeof val === 'string') {
+            return `'${val}'`;
+          }
+          if (separator === '=' && typeof val !== 'object') {
+            return `{${val}}`;
+          }
+          return val;
+        };
+
+        const value = obj[key];
+        if (typeof value === 'object' && !Array.isArray(value)) {
+          return `${indent}${key}${separator}${separator === '=' ? '{' : ''}{\n${formatObject(
+            getValue(value),
+            indentLevel + 1,
+            ': ',
+          )}\n${indent}${separator === '=' ? '}' : ''}}`;
+        }
+
+        if (Array.isArray(value)) {
+          return `${indent}${key}${separator}${separator === '=' ? '{' : ''}[${value.map((val) =>
+            getValue(val),
+          )}]${separator === '=' ? '}' : ''}`;
+        }
+
+        return `${indent}${key}${separator}${getValue(value)}`;
+      })
+      .join('\n');
+  }
+
+  if (!componentProps) {
+    return '';
+  }
+
+  return `\n${formatObject(componentProps, spacing, '=')} `;
+}
+
+const getCodeExample = ({
+  selectedDemo,
+  selectedSlot,
+  selectedCustomizationOption,
+  selectedTokens,
+  componentName,
+  examples,
+  theme,
+}: Props) => {
+  const tokens = {
+    ...selectedTokens,
+    borderColor: DEFAULT_COLORS[selectedTokens.color][500],
+    border: `${selectedTokens.borderWidth}px solid`,
+    backgroundColor:
+      theme.palette.mode === 'light'
+        ? DEFAULT_COLORS[selectedTokens.color][100]
+        : DEFAULT_COLORS[selectedTokens.color][700],
+    color:
+      theme.palette.mode === 'light'
+        ? DEFAULT_COLORS[selectedTokens.color][700]
+        : DEFAULT_COLORS[selectedTokens.color][100],
+  };
+
+  const getTokensString = (indent: number = 0): string => {
+    const spaces = ' '.repeat(indent * 2);
+    return (Object.keys(tokens) as Array<keyof typeof tokens>).reduce((acc, key) => {
+      return `${acc}\n${spaces}${key}: ${
+        typeof tokens[key] === 'string' ? `'${tokens[key]}'` : tokens[key]
+      },`;
+    }, '');
+  };
+
+  let code = examples?.comments ? `\n// ${examples?.comments}` : '';
```

Comment:
It would be great to wrap the comment to place it on two lines here:

<img width="767" alt="Screenshot 2023-09-17 at 19 44 20" src="https://github.com/mui/mui-x/assets/3165635/f363e469-a5f9-4d24-8b7d-2baa5a9125f2">

otherwise people have to scroll.